From patchwork Mon Mar  7 12:44:11 2016
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [U-Boot] sunxi: fix DCDC2 output in CHIP_defconfig
From: Maxime Ripard <maxime.ripard@free-electrons.com>
X-Patchwork-Id: 592940
Message-Id: <1457354651-10083-1-git-send-email-maxime.ripard@free-electrons.com>
To: Hans de Goede <hdegoede@redhat.com>
Cc: u-boot@lists.denx.de
Date: Mon,  7 Mar 2016 13:44:11 +0100

From: Boris Brezillon <boris.brezillon@free-electrons.com>

Unlike the datasheet recommendation, the R8 SoC requires a 1.4V supply
for its CPU when operating at 1Ghz.

Rely on the default value specified in the Kconfig entry.

Signed-off-by: Boris Brezillon <boris.brezillon@free-electrons.com>
Signed-off-by: Maxime Ripard <maxime.ripard@free-electrons.com>
---
 configs/CHIP_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/CHIP_defconfig b/configs/CHIP_defconfig
index 78b2c511bfa2..3135d1c88e79 100644
--- a/configs/CHIP_defconfig
+++ b/configs/CHIP_defconfig
@@ -9,7 +9,6 @@ CONFIG_SPL=y
 CONFIG_SYS_EXTRA_OPTIONS="CONS_INDEX=2"
 # CONFIG_CMD_IMLS is not set
 CONFIG_CMD_GPIO=y
-CONFIG_AXP_DCDC2_VOLT=1300
 CONFIG_AXP_ALDO3_VOLT=3300
 CONFIG_AXP_ALDO4_VOLT=3300
 CONFIG_USB_MUSB_GADGET=y
